packages/latino/build.sh

```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://www.lenguajelatino.org/
 TERMUX_PKG_DESCRIPTION="Lenguaje de programación de código abierto para latinos y de habla hispana."
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1.3.0
+TERMUX_PKG_VERSION=1.3.1
 TERMUX_PKG_REVISION=1
```

Comment:
No need of this because package didn't pushed on repo yet.

Reply:
Yes, please remove this line.